content/blogs/newsletter-104.md

```diff
@@ -0,0 +1,90 @@
+While the development continued and still continues as usual, the development team hadn't seen any enlargement with potential developers.
+In worst cases some of the existing core developers became inactive for quite a while, of which were maintainers of important core modules of the project.
+
+This left the development workflow in a state of paralysis for some of the module components of ReactOS, as nobody else in the team had the right skills to determine the correctness of the patches provided
+by contributors in order to get them shipped in [master](https://github.com/reactos/reactos) branch.
+The ReactOS Team is currently adopting an approach to circumvent this problem, by enrolling contributors to the development team who are known to have contributed with useful patches for at least a year, have enough coding skills, have collaborated with the project team members and are willing to work on the main field of area they are interested in.
+
+With that said, the ReactOS Team is pleased to announce the upcoming of 4 new developers for the project -- Carl J. Bialorucki (cbialorucki), Doug Lyons (Doug-Lyons), Oleg Dubinskiy (oleg-dubinskiy) and Whindmar Saksit (whindsaks).
+Their main focus is to maintain the Shell and Audio components respectively, as well as on any module that requires assistance.
+
+## Symmetric Multiprocessing (SMP) support status & GCC 13 preparations
+The symmetric multiprocessing support for AMD64 architecture is getting extensively improved thanks to Timo Kreuzer (tkreuzer).
+In particular, he has implemented code for processor freezing and switching ([PR #6581](https://github.com/reactos/reactos/pull/6581)) so that every logical processor is frozen or switched properly.
+In addition he implemented basic support for IPIs (interprocessor interrupts) which are needed for SMP scheduling, debugging and whatnot ([PR #6130](https://github.com/reactos/reactos/pull/6130)).
+These are only limited to APC, DPC and freeze IPIs for the moment being.
+
+Another important improvement in the area of SMP is the kernel initialization in the AMD64 build architecture, which got refurbished by Timo ([PR #6110](https://github.com/reactos/reactos/pull/6110)).
+This allows the SMP support to work properly.
+
+Timo also helped fixing some bugs in order to get ReactOS compile-able for GCC 13 build environment.
+Most of the chunk of GCC 13 preparations goes thanks to Simone Mario Lombardo (simonelombardo) for making ReactOS bootable.
+
+In the meantime Justin Miller (The_DarkFire) is focusing on the x86 architecture of the SMP area.
+He worked on the preparations for the application processor bootup ([PR #5879](https://github.com/reactos/reactos/pull/5879)) in x86 SMP HAL and is planning to further improve the x86 part of SMP.
+
+**It is worth noting that SMP is still heavily under development and there is no exact estimation when will the multicore support finally come out as ready, so stay tuned for more updates!**
```

Comment:
```suggestion
![IMG_4862](https://github.com/reactos/web-content/assets/23200630/7118dc0d-5975-4db5-a9b2-bafa8baa3db0)


**It is worth noting that SMP is still heavily under development and there is no exact estimation when will the multicore support finally come out as ready, so stay tuned for more updates!**
```

Reply:
Suggested Change of:
"no exact estimation when will the multicore support finally out as ready,"
To:
"no exact estimate of when the multi-core support will finally be ready,"